main.py

```
@@ -54,6 +100,9 @@ def run(self):
     
     def _crawl_and_convert(self):
         """クロールと変換を統合実行"""
+        import time
```

Ruff (0.11.9)

Redefinition of unused `time` from line 8

Remove definition: `time`

(F811)